src/unix/web/matoya.js

```diff
@@ -749,6 +752,8 @@ async function mty_ws_connect(url) {
 		ws.onmessage = (ev) => {
 			ws.msgs.push(ev.data);
 			Atomics.notify(ws.sync, 0, 1);
+			tm = setInterval(() => {ws.send('__ping__');}, 60000); // we've seen timeouts at the 10-minute mark,
+			                                                       // so a 60-second period should be more than enough
```

Comment:
Shouldn't this be in `ws.onopen`? Since won't this create a new interval every time a message gets in? https://developer.mozilla.org/en-US/docs/Web/API/setInterval